packages/global/scss/components/blocks/_hero-card.scss

```diff
@@ -48,7 +48,12 @@
         @include skin-typography($style: "slug-small", $link-style: "primary");
       }
     }
-    &--hero {
```

Comment:
It looks like the `hero` modifier is used in several places, have they all been updated?